Sources/FioriSwiftUICore/_FioriStyles/TimelineStyle.fiori.swift

```diff
@@ -0,0 +1,220 @@
+        }
+    }
+
+    struct SecondaryTimestampLabelFioriStyle: SecondaryTimestampLabelStyle {
+        let timelineConfiguration: TimelineConfiguration
+        @Environment(\.isEnabled) var isEnabled: Bool
+
+        func makeBody(_ configuration: SecondaryTimestampLabelConfiguration) -> some View {
+            SecondaryTimestampLabel(configuration)
+                .font(.fiori(forTextStyle: .caption1))
+                .foregroundColor(self.isEnabled ? Color.preferredColor(.primaryLabel) : Color.preferredColor(.separator))
+        }
+    }
+
+    struct SecondaryTimestampImageFioriStyle: SecondaryTimestampImageStyle {
+        let timelineConfiguration: TimelineConfiguration
+        @Environment(\.isEnabled) var isEnabled: Bool
+
+        func makeBody(_ configuration: SecondaryTimestampImageConfiguration) -> some View {
+            SecondaryTimestampImage(configuration)
+                .foregroundColor(self.isEnabled ? Color.preferredColor(.tertiaryLabel) : Color.preferredColor(.separator))
+        }
+    }
+
+    struct UpperVerticalLineFioriStyle: UpperVerticalLineStyle {
+        let timelineConfiguration: TimelineConfiguration
+    
+        func makeBody(_ configuration: UpperVerticalLineConfiguration) -> some View {
+            UpperVerticalLine(configuration)
+                .frame(width: 2, height: 16)
+                .background(self.timelineConfiguration.isPast || self.timelineConfiguration.isPresent ? Color.preferredColor(.tintColor) : Color.preferredColor(.grey3))
+        }
+    }
+    
+    struct NodeImageFioriStyle: NodeImageStyle {
+        let timelineConfiguration: TimelineConfiguration
+
+        func makeBody(_ configuration: NodeImageConfiguration) -> some View {
+            NodeImage(configuration)
+                .foregroundColor(self.timelineConfiguration.isPast || self.timelineConfiguration.isPresent ? Color.preferredColor(.tintColor) : Color.preferredColor(.separatorOpaque))
+                .font(.system(size: 15, weight: .bold))
+        }
+    }
+
+    struct LowerVerticalLineFioriStyle: LowerVerticalLineStyle {
+        let timelineConfiguration: TimelineConfiguration
+    
+        func makeBody(_ configuration: LowerVerticalLineConfiguration) -> some View {
+            LowerVerticalLine(configuration)
+                .frame(width: 2)
+                .background(self.timelineConfiguration.isPast || self.timelineConfiguration.isPresent ? Color.preferredColor(.tintColor) : Color.preferredColor(.grey3))
+        }
+    }
+
+    struct TitleFioriStyle: TitleStyle {
+        let timelineConfiguration: TimelineConfiguration
+        @Environment(\.isEnabled) var isEnabled: Bool
+
+        func makeBody(_ configuration: TitleConfiguration) -> some View {
+            Title(configuration)
+                .font(.fiori(forTextStyle: .headline))
+                .foregroundColor(self.isEnabled ? Color.preferredColor(.primaryLabel) : Color.preferredColor(.separator))
```

Comment:
```suggestion
                .foregroundColor(self.isEnabled ? Color.preferredColor(.primaryLabel) : Color.preferredColor(.quaternaryLabel))
```
I think UIKit uses quaternaryLabel when disabled.

Reply:
Updated all disabled text color to quaternaryLabel.